QUICKSTART.md

```diff
@@ -6,57 +6,29 @@ category: quick start
 
 # TiDB Quick Start Guide
 
-This guide introduces how to deploy and monitor a TiDB cluster on your local drive using Docker Compose for experimenting and testing.
+Installation instructions for platforms officially supported by PingCAP.
 
-> **Warning:** Deploying TiDB using Docker Compose can only be used for experimental purposes. For production usage, [use Ansible to deploy the TiDB cluster](op-guide/ansible-deployment.md).
+- [Ansible (recommended)](op-guide/ansible-deployment.md)
+- [Ansible (offline deployment)](op-guide/offline-ansible-deployment.md)
+- [Docker](op-guide/docker-deployment.md)
+- [Docker Compose (development environments)](op-guide/docker-compose.md)
+- [Kubernetes (beta)](op-guide/kubernetes.md)
 
-## Prerequisites
+## Community Provided Blog Posts & Tutorials
 
-Before you begin, make sure to install the following tools:
+A collection of unofficial installation guides for TiDB authored by the community. The content is not guaranteed to be up to date.
 
-- [Git](https://git-scm.com/downloads)
-- [Docker Compose](https://docs.docker.com/compose/install/)
-- [MySQL Client](https://dev.mysql.com/downloads/mysql/)
+- [How To Spin Up an HTAP Database in 5 Minutes with TiDB + TiSpark](https://www.pingcap.com/blog/how_to_spin_up_an_htap_database_in_5_minutes_with_tidb_tispark/)
+- [Developer install guide (single machine)](http://www.tocker.ca/this-blog-now-powered-by-wordpress-tidb.html)
 
-## Deploy a TiDB cluster
+_Please open a [pull request](https://github.com/pingcap/docs/edit/master/QUICKSTART.md) to add additional links._
```

Comment:
Your contribution is also welcome! Feel free to open a [pull request](https://github.com/pingcap/docs/edit/master/QUICKSTART.md) to add additional links.